UnityProject/Assets/Editor/LevelBuilder/GUI/LevelBuilderWindow.cs

```diff
@@ -19,16 +19,24 @@
 
 public class LevelBuilderWindow : EditorWindow
 {
+    public static float SectionTitleHeight = 20;
```

Comment:
Should this not be in `OverlayMenu()` instead? Confusing.

Reply:
I need to change the other menus to use this